src/components/error/_macro-options.md

```diff
@@ -0,0 +1,5 @@
+| Name       | Type                      | Required | Description                                                        |
+| ---------- | ------------------------- | -------- | ------------------------------------------------------------------ |
+| id         | string                    | true     | The HTML id attribute that the error summary link can anchor to    |
+| text       | string or array`<string>` | true     | The text describing the error                                      |
+| attributes | object                    | false    | HTML attributes (for example, data attributes) to add to the field |
```

Comment:
We dont want this file added as error isnt really a component the user can use on its own. Because of this, this has been documented in the content on the service manual